[MTM-66200] Updated docs for HTML widget (#4563)
SoftwareAG/c8y-docs: content/cockpit/widgets-collection.md

**Parameters to configure**

* **Asset selection**: Optionally, select the asset whose managed object will be accessible via `c8yContext` expressions.

> * **Asset properties**: In the **Asset properties** section, you can add mappings to properties. Click **Add mappings**. Then select an asset and select properties from **Asset properties**, **Custom properties**, and **Computed properties** tabs. Use the icon buttons to copy a code expression, clear a mapping, assign another property, or remove a mapping. At runtime, all mapped values are accessible in the HTML code under `c8yProperties`, a plain object keyed by the mapping names. If you rename a mapping key, the references in the HTML code will be updated.

Reviewer: ```suggestion
* **Asset properties**: In the **Asset properties** section, you can add mappings to properties. Click **Add mappings**. Then select an asset and select properties from the **Asset properties**, **Custom properties**, and **Computed properties** tabs. Use the icon buttons to copy a code expression, clear a mapping, assign another property, or remove a mapping. At runtime, all mapped values are accessible in the HTML code under `c8yProperties`, a plain object keyed by the mapping names. If you rename a mapping key, the references in the HTML code will be updated.
```

**Parameters to configure**

* **Target assets or devices**: Select the objects for which optional HTML expressions are evaluated.
* **Asset properties**: In the **Asset properties** section, you can add mappings to asset properties. Click **Add mappings**, then select an asset, then select an asset/regular/computed property you'd like to use in the HTML code. Use icon buttons to copy a code expression, clear a mapping, assign another property, or remove a mapping. When you rename a mapping key, all references in the HTML code are updated automatically. At runtime, all mapped values are accessible in the HTML code under `this.c8yProperties`, a plain object keyed by the mapping names.

> 

Author: ```suggestion
* **Asset properties**: In the **Asset properties** section, you can add mappings to properties. Click **Add mappings**. Then select an asset and select properties from **Asset properties**, **Custom properties**, and **Computed properties** tabs. Use the icon buttons to copy a code expression, clear a mapping, assign another property, or remove a mapping. At runtime, all mapped values are accessible in the HTML code under `this.c8yProperties`, a plain object keyed by the mapping names. If you rename a mapping key, the references in the HTML code will be updated.
```

The widget offers two distinct modes:

1. **Normal mode**: You can apply HTML and CSS while adding properties as template literals. You can use simple expressions such as:

>    `${this.c8yContext ? this.c8yContext?.name : 'No device selected'}`. The `${this.c8yContext}` variable always refers to the selected target asset. Use `${this.c8yProperties?.<mappingName>}` to access mapped asset properties.

Author: ```suggestion
   - `${this.c8yContext ? this.c8yContext?.name : 'No device selected'}` where `c8yContext` variable always refers to the widget's selected asset
   - `${this.c8yProperties?.<mappingName>}` to access values of the mapped asset properties
   - always use optional chaining `?.` because these objects may be `undefined` while data is loading
```
